llvm/test/MC/Disassembler/X86/apx/evex-format.txt

```diff
@@ -11,6 +11,12 @@
 # INTEL: add	r18, qword ptr [r17 + 123], r16
 0x62,0xec,0xec,0x10,0x01,0x41,0x7b
 
+## MRMDestMemCC
+
+# ATT:   cfcmovbq	%r9, 123(%r8,%rax,4)
+# INTEL: cfcmovb	qword ptr [r8 + 4*rax + 123], r9
+0x62,0x54,0xfc,0x0c,0x42,0x4c,0x80,0x7b
```

Comment:
Please use r16, r17, r18 so that we check the prefix is set correctly. Same for others.